types/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import { RequestHandler } from "express";
+import https from 'https';
+import jose from 'jose';
+
+declare namespace errors {
+    declare class InsufficientScopeError extends Error {
+        constructor();
+    }
+    declare class TokenClaimsError extends Error {
+        constructor();
+    }
+}
+
+interface Claim {
+    name: string;
+    value?: string;
+}
+
+interface Options {
+    claims?: Claim[];
+    scope?: string[];
+}
+
+interface JwksService {
+    (callback: any, refreshKeyStore?: boolean):any
```

Comment:
We can be more specific here. `callback` is function: `callback: (error?: Error, keyStore?: KeyStore) => void` and the whole method returns `Promise<void>`

Reply:
There's also a `;` missing at the end. I know it's not required but I think we should be consistent across the whole file.